contracts/container/doc.go

```diff
@@ -104,6 +104,8 @@ Key-value storage format:
    NNS root domain zone for containers
  - 'x<cid>' -> []byte
    container descriptors encoded into NeoFS API binary protocol format
+ - '0<cid>' -> std.Serialize(Info)
```

Comment:
`0` is ambiguous here in that all other values in the list are characters while this one is a 0x00 byte, not 0x30. I'd suggest using a character to follow the scheme we already have.